src/game_character.cpp

```diff
@@ -810,7 +809,7 @@ void Game_Character::CancelMoveRoute() {
 }
 
 int Game_Character::GetTileId() const {
-	return tile_id;
+	return GetSpriteName().empty() ? GetSpriteIndex() : 0;
 }
```

Comment:
> Simplify GetTileId() 
should be tested against this test case: https://github.com/EasyRPG/Player/pull/1179